src/fides/api/ops/service/connectors/__init__.py

```diff
@@ -14,6 +14,9 @@
 from fides.api.ops.service.connectors.email_connector import (
     EmailConnector as EmailConnector,
 )
+from fides.api.ops.service.connectors.fides_connector import (
+    FidesConnector as FidesConnector,
+)
```

Comment:
🤦

Reply:
well, just to play devil's advocate - all of the other imports have the `as` even with the name duplicated ... so that's why i just decided to follow convention. any idea why we're doing it this way? i don't mind changing it, i agree that it seems unnecessary...

Reply:
I'm not sure why it would be that way. It obviously isn't hurting anything and still works, just unnecessary.

Reply:
i think i see why this is being done now. when i remove it, i get the following mypy error:
```
root@284a2f6d5653:/fides# mypy
src/fides/api/ops/task/task_resources.py:16: error: Module "fides.api.ops.service.connectors" does not explicitly export attribute "FidesConnector"; implicit reexport disabled  [attr-defined]
    from fides.api.ops.service.connectors import (
    ^
Found 1 error in 1 file (checked 291 source files)
```

So it seems like this is being done to allow all the connector classes to be then be imported from this module directly (as is done in `task_resources.py`)

Reply:
ah yes, I forgot about this reexport error.